x/auth/tx/config/config.go

```diff
@@ -38,9 +38,12 @@ func init() {
 
 type ModuleInputs struct {
 	depinject.In
-	Config              *txconfigv1.Config
-	ProtoCodecMarshaler codec.ProtoCodecMarshaler
-	ProtoFileResolver   txsigning.ProtoFileResolver
+
+	Config                *txconfigv1.Config
+	AddressCodec          address.Codec
+	ValidatorAddressCodec runtime.ValidatorAddressCodec
```

Comment:
dont think we need this here

Reply:
We do, it is used right below in the tx config options.